examples/observability_at/version.tf

```diff
@@ -6,5 +6,9 @@ terraform {
       source  = "ibm-cloud/ibm"
       version = "1.49.0"
     }
+    logdna = {
+      source  = "logdna/logdna"
+      version = "1.13.1"
```

Comment:
will need to be locked into 1.14.2 after updating in version.tf

Reply:
Fixed